advocacy_docs/repos/getting_started/with_web/get_your_token.mdx

```diff
@@ -4,29 +4,27 @@ navTitle: Get your token
 description: How to get the token you need to download EDB software
 ---
 
-## Tokens and EDB Accounts
+## Tokens and EDB accounts
 
 To download EDB software, you need a token. 
 This token is a string that identifies you and your eligibility to download software from EDB. 
-If you have an account on EDB, your already have a token. 
-We'll show you how to retrieve it.
+If you have an account on EDB, you already have a token and can retrieve it.
```

Comment:
```suggestion
If you have an account on EDB, you already have a token and which you can retrieve from your account .
```

Reply:
Your grammar has gone all wonky in this suggestion @djw-m, but I see the intent. Assertions like "you already have a token and can retrieve it" can be read as presumptuous/terse/dismissive.  We need to signpost that we're about to explain _how_ to do this. Perhaps "retrieve it _by following these steps_" or something.